cfme/tests/cloud/test_quota.py

```diff
@@ -298,3 +305,195 @@ def test_project_quota_enforce_via_lifecycle_cloud(
         provision_request.wait_for_request(method="ui")
         request.addfinalizer(provision_request.remove_request)
         assert provision_request.row.reason.text == "Quota Exceeded"
+
+
+@pytest.fixture
+def admin_email(appliance):
+    """Required for user quota tagging services to work, as it's mandatory for it's functioning."""
+    user = appliance.collections.users
+    admin = user.instantiate(name='Administrator')
+    with update(admin):
+        admin.email = fauxfactory.gen_email()
+    yield
+    with update(admin):
+        admin.email = ''
+
+
+@pytest.fixture(scope="module")
+def automate_flavour_method(appliance, klass, namespace):
```

Comment:
consistent spelling please - `automate_flavor_method`, we use american-ized spelling otherwise (as does MIQ)